Library/Homebrew/cmd/update.rb

```diff
@@ -108,7 +108,8 @@ def update
 
       begin
         f = Formulary.factory("#{user}/#{repo}/#{newname}")
-      rescue FormulaUnavailableError, *FormulaVersions::IGNORED_EXCEPTIONS
+      # short term fix to prevent situation like https://github.com/Homebrew/homebrew/issues/45616
+      rescue Exception
```

Comment:
Should probably also handle `RuntimeError` with just a `rescue` if you go this route?